cmd/csi/provisioner/builder.go

```diff
@@ -113,7 +115,7 @@ func (builder CommandBuilder) buildRun() func(*cobra.Command, []string) error {
 		}
 
 		signalHandler := ctrl.SetupSignalHandler()
-		otelShutdownFn := otel.Start(signalHandler, "dynatrace-csi-provisioner", csiManager.GetAPIReader(), builder.namespace)
+		otelShutdownFn := dtotel.Start(context.Background(), "dynatrace-csi-provisioner", csiManager.GetAPIReader(), builder.namespace)
```

Comment:
why are you not using the `signalHandler`?
we should have 1 context that is used by everything, no?

the `signalHandler` is just a "fancy" context that gets cancelled in case of a sigterm or sigkill.

Reply:
fixed